mlflow/transformers/__init__.py

```diff
@@ -513,6 +516,18 @@ def save_model(
     if mlflow_model is None:
         mlflow_model = Model()
 
+    if task and task.startswith(_LLM_INFERENCE_TASK_PREFIX):
```

Comment:
Note: Previously we determine the task is `llm/v1` inference task or Transformer's build-in task, by using Transformers internal API that lists all supported task types ([code](https://github.com/mlflow/mlflow/blob/master/mlflow/transformers/__init__.py#L1461-L1466)). But just checking our task name is simpler here.